src/utils/payoutConfigByNetworkId.ts

```diff
@@ -1,19 +1,13 @@
 // available tokens for payouts
-export const PAYMENT_TOKEN_PER_NETWORK: Record<string, { rpc: string; token: string; decimals: number }> = {
+export const PAYMENT_TOKEN_PER_NETWORK: Record<string, { rpc: string; }> = {
```

Comment:
Since this is just an RPC source we could use `rpc-handler` here to add a bit of redundancy and allows it to scale with other chains. 

I'd think that might be out of scope for this PR but maybe as another issue?

Reply:
Just saw the issue you opened